velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +485,119 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, weekDate) {
+  const auto week = [&](std::optional<Date> date) {
+    return evaluateOnce<int64_t>("week(c0)", date);
+  };
+  auto parseDate = [&](const StringView& dateStr) {
+    Date returnDate;
+    parseTo(dateStr, returnDate);
+    return returnDate;
+  };
+
+  EXPECT_EQ(1, week(parseDate("1919-12-31")));
```

Comment:
consider refactoring to allow for shorter code: `EXPECT_EQ(1, week("1919-12-31"));`